.github/workflows/build-and-publish.yml

```
@@ -0,0 +1,263 @@
+          if find . \( -name "*.Tests.csproj" -o -name "*Tests.csproj" \
+          -o -name "*Test.csproj" \) | grep -q .; then
+            dotnet test --configuration Release --no-build --verbosity normal
+          else
+            echo "No traditional test projects found, skipping unit tests"
+          fi
+
+      - name: Run SDK functional tests
+        run: |
+          echo "Running Bento SDK functional tests..."
+          cd Bento.Examples
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)